ayon_api/server_api.py

```diff
@@ -6068,26 +6079,117 @@ def get_product_type_names(
             )
         }
 
+    def get_product_base_types(
+        self, fields: Optional[Iterable[str]] = None
+    ) -> List["ProductBaseTypeDict"]:
+        """Types of product base types.
+
+        Args:
+            fields (Optional[Iterable[str]]): Product base types fields
+                to query.
+
+        Returns:
+            list[ProductBaseTypeDict]: Product base types information.
+
+        """
+        if not fields:
+            fields = self.get_default_fields_for_type("productBaseType")
+
+        query = product_types_query(fields)
+
+        parsed_data = query.query(self)
+
+        return parsed_data.get("productBaseTypes", [])
+
+
+    def get_project_product_base_types(
+        self,
+        project_name: str,
+        fields: Optional[Iterable[str]] = None
+    ) -> List["ProductBaseTypeDict"]:
+        """Product base types available in a project.
+
+        Filter only product base types available in a project.
+
+        Args:
+            project_name (str): Name of the project where to look for
+                product base types.
+            fields (Optional[Iterable[str]]): Product types fields to query.
+
+        Returns:
+            List[ProductBaseTypeDict]: Product Base types information.
+
+        """
+        if not fields:
+            fields = self.get_default_fields_for_type("productBaseType")
+
+        query = project_product_base_types_query(fields)
+        query.set_variable_value("projectName", project_name)
+
+        parsed_data = query.query(self)
+
+        return parsed_data.get("project", {}).get("productBaseTypes", [])
+
+
+    def get_product_base_type_names(
```

Comment:
This function probably does not have to be implemented in server api.

It does 2 different things, either returns what `get_project_product_base_types` does or what `get_products` does. Looks like a helper functions somewhere in pipeline, but I would not include it here.